### PR TITLE
Blog Post: OpenSats' 2023 Year in Review

### PR DESCRIPTION
**Pull request by dergigi**
To be reviewed and sanity-checked.

**Build preview:** 

- [/blog/2023-year-in-review](https://os-website-git-eoy-2023-open-sats.vercel.app/blog/2023-year-in-review)

**Social preview:**

Screenshot 2023-12-31 at 16 57 05

### CODE REVIEW

File: data/blog/2023-year-in-review.mdx [hunk 1]
```diff
@@ -0,0 +1,402 @@
...
+<center>
+    <DonateToNostrFundButton/>
+</center>
+
+---
+
+## Grantees
+
+After the initial wave of grants announced in July, we’ve added around 10
+grantees each month on average, totalling 90 approved grants as of this writing
```
Comment (elaineo) [inline data/blog/2023-year-in-review.mdx:351]: "totaling"

File: data/blog/2023-year-in-review.mdx [hunk 2]
```diff
@@ -0,0 +1,402 @@
...
+Most of our grants are being paid out monthly. In sum we've sent more than seven
+billion sats to our grantees, in addition to a \$1M grant given to
+[GrapheneOS](/projects/grapheneos) at the beginning of the
+year. We will continue to support bitcoin-adjacent projects like Graphene and
+[The Tor Project](/projects/tor), as many people have to
+rely on these tools to use bitcoin in a secure and self-sovereign way.
+
+We expect to have over 100 grantees by Q2 of 2024, given past growth and pending
+grant applications. We hope to support many of our grantees for the long-term,
+as long as mission-aligned progress and future funding is a given.
```
Comment (elaineo) [inline data/blog/2023-year-in-review.mdx:371]: passive voice, maybe change to "as long as future funding permits and progress is aligned with our mission"

File: data/blog/2023-year-in-review.mdx [hunk 3]
```diff
@@ -0,0 +1,402 @@
...
+[Krux](https://github.com/selfcustody/krux), and
+[Vexl](https://github.com/vexl-it/vexl).
+
+We are constantly processing and evaluating applications, and we will announce
+more grants that are sourced from the General Fund soon.
+
+### Nostr: The Nostr Fund
+
+In 2023, our main focus for The Nostr Fund was to support projects that make
+Nostr more accessible, secure, decentralized, and easy to use. We also provided
```
Comment (elaineo) [inline data/blog/2023-year-in-review.mdx:268]: *nostr (lowercase for consistency)

**Review by elaineo — APPROVED** [review]
minor edits. looks great & i love the social media header

**Review by Arvin21M — APPROVED** [review]
🚀 🚀 🚀 🚀 🚀 🚀 🚀 🚀 🚀 🚀 🚀 🚀 

What a year. Bravo to the entire OpenSats fam.

- All links work and point to the right place. 
- The entire article reads really well to me. 
- Noted three areas of consideration for edits/changes, but not something absolutely required, more nice to have. 

See screenshots with notes:

---

image

---

image

---

image

**Comment by cfunk7777** [comment]
contrast looks fine on my browser as the background is black (not white as yours @Arvin21M, for some reasons)
Capture d’écran, le 2023-12-31 à 18 41 05


> 
> image

**Comment by cfunk7777** [comment]
it's perfect! the only thing it leaves me hungry for is a link to `our domain-expert committees` but as Arvin said above, it's just a nice-to-have if the link is already readily available

**Comment by dergigi** [comment]
Thanks for the edits @elaineo and thanks for the review @Arvin21M  🙏🧡

Let's fire it out, I won't be able to make the image adjustments before the year actually ends 😅

Thanks everyone! Love ya'll 🧡

**Comment by bitcoinprecept** [comment]
Looks great

**Pull request merged**